go-controller/pkg/node/node.go

```diff
@@ -706,8 +706,13 @@ func (n *OvnNode) checkAndDeleteStaleConntrackEntriesForNamespace(newNs *kapi.Na
 		go func(gwIP string) {
 			defer wg.Done()
 			if len(gwIP) > 0 {
+				// TODO: Implement IPv6
 				if hwAddr, err := util.GetMACAddressFromARP(net.ParseIP(gwIP)); err != nil {
-					klog.Errorf("Failed to lookup hardware address for gatewayIP %s: %v", gwIP, err)
+					if strings.Contains(gwIP, ":") { //checks if gwIp is an IPv6 address
```

Comment:
this is not a good way to figure out if an address is v6 or v4. Please use https://pkg.go.dev/k8s.io/utils/net#IsIPv6String